File: C/1412.c
#include <stdio.h>
#include <string.h>

int main(){
	char inStr[90];
	gets(inStr);
	int alpha[26];
	for(int i = 0; i < 26; i++){
		alpha[i] = 0;
	}
	
	for(int j = 0; j < strlen(inStr); j++){
		for(int i = 97; i <= 122; i++){
			if(inStr[j]==i){
				alpha[i - 97]++;
			}
		}
	}
	
	for(int i = 97; i <= 122; i++){
		printf("%c:%d\n", i, alpha[i - 97]);
	}
	return 0;
}

File: Python/1412.py
import string

inStr = input()
alpha = list(string.ascii_lowercase)
alphaCnt = [0] * 26

for i in inStr:
    for j in range(26):
        if i==alpha[j]:
            alphaCnt[j] += 1

for i in range(26):
    print("%c:%d" % (alpha[i], alphaCnt[i]))